### DISCUSSION TITLE
What is the largest lottery 💰💰 ever won?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#874] [Galaxy Trivia] [answered]
**Bonus:** What lotto number has come up most? (

*reported by LottoNZ*

**Comment by peterbaccaro** [comment, marked as answer]
The largest lottery jackpot ever won was a record-breaking **$2.05 billion (£1.79 billion)** prize.  It was won by California resident Edwin Castro in November 2022.

https://www.guinnessworldrecords.com/news/2023/2/winner-of-biggest-ever-lottery-jackpot-revealed-after-bagging-2-billion-ticket-738096

According to LottoNZ, the number **1** has come up the most.

https://mylotto.co.nz/game-information
